save-orchestrator/build.gradle.kts

```diff
@@ -23,20 +23,29 @@ tasks.withType<KotlinCompile> {
     }
 }
 
-// if required, file can be provided manually
-@Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
+val downloadSaveCliDistribTaskProvider: TaskProvider<Download> = tasks.register<Download>("downloadSaveCliDistrib") {
+    src(KotlinClosure0(function = { findProperty("saveCliDistribFilepath") ?: "file:\\\\" }))
+    File("$buildDir/libs/").mkdirs()
+    dest("$buildDir/libs")
+    overwrite(false)
+}
+
+downloadSaveCliDistribTaskProvider.configure {
+    enabled = findProperty("saveCliDistribFilepath") != null
```

Comment:
I think `enabled` can be set inside `register`?

Reply:
fixed